main/guides/getting-started/README.md

```diff
@@ -1,163 +1,199 @@
+- **docker-compose.yml** - for running a local Agoric blockchain
 
-## Install Git
+::: tip TODO: prune api, agstate
 
-Follow [Git installation instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) or use your platform's package manager.
+Disregard these contents, for now:
 
-```shell
-npm install --global git
-git --version # 2.25.0 or higher
+- **\_agstate**
+- **api**
+  :::
+
+      node_modules
+
+## Install dependencies
+
+The UI depends on the React framework, and the contract depends on
+the Agoric framework. The packages in this project also have
+development dependencies for testing, code formatting, and static analysis.
+
+```sh
+cd demo
+yarn install  # may take several minutes
 ```
 
-## Clone the Agoric SDK
+::: tip Troubleshooting `yarn install`
 
-```shell
-git clone --branch community-dev https://github.com/Agoric/agoric-sdk
-cd agoric-sdk
+If you run into errors during install or build, make sure you have the relevant developer tools installed. For example, on Debian or Ubuntu Linux, you can run `sudo apt get install build-essential` to install these tools.
+
+Also, check that you are on a
+[supported platform](#platform-linux-shell-or-equivalent) and
+not native Windows.
+:::
+
+## Start a local Agoric blockchain
+
+To run a local Agoric blockchain with [docker-compose](https://docs.docker.com/compose/):
+
+```sh
+yarn start:docker
 ```
 
-Cloning and installing the Agoric SDK can take a while. Please be patient.
+::: tip Note: large docker image
 
-## Install NPM Dependencies
+The docker image used here is several gigabytes.
+The fist time you pull it may take several minutes.
 
-```shell
-yarn install
+:::
+
+Look at the logs from the blockchain until it is steadily making blocks:
+
+```sh
+yarn docker:logs
 ```
 
-**Note:** If you run into errors during install or build, make sure you have the relevant developer tools installed. For example, on Debian or Ubuntu Linux, you can run `sudo apt get install build-essential` to install these tools.
+## Deploy the Contract and Start the UI
 
-## Build Packages
+Let's start it up:
 
-```shell
-yarn build
+```sh
+yarn start
 ```
 
-**Note:** If this `yarn build` step fails, check that you are on a
-[supported platform](#platform-linux-shell-or-equivalent) and
-not native Windows.
+::: tip TODO: integrated `yarn start` script
 
-## Build the Cosmic Swingset Package
+For now, use the [Agoric Gov Proposal Builder](https://cosgov.org/)
+for deployment:
 
-```shell
-(cd packages/cosmic-swingset && make)
+1. Make the contract and proposal bundles.
+
+```sh
+(cd contract; yarn build:proposal)
 ```
 
-## Install Agoric CLI
+Note the long `b1-xxx.json` bundle filenames
+as well as `start-game1-permit.json`
+and `start-game1.js`.
 
-Use `yarn link-cli` to install the Agoric CLI (Command Line Interface) in a convenient place of your choosing such as:
+2. Use the [Install Bundle](https://cosgov.org/?msgType=installBundle&network=local) tab to install the 2 bundles.
```

Comment:
This has been failing lately:

 - https://github.com/DCFoundation/cosmos-proposal-builder/issues/12

@kbennett2000 and @sufyaankhan ran into this, I think. or was it @toliaqat ? or both?

Reply:
We have some contributions for an [single yarn start command](https://github.com/agoric-labs/dapp-game-places/issues/5) that would avoid this.

Reply:
The "Request body too large" error doesn't look like a cosgov bug but rather a zoe packaging bug:

 - https://github.com/Agoric/agoric-sdk/issues/8621

Meanwhile, please take a look at the recent update using a single `yarn start:contract` command instead of cosgov.